raaghu-elements/src/rds-tab-group/rds-tab-group.stories.tsx

```
@@ -37,11 +37,11 @@ const meta: Meta =  {
       control: { type: "select" },
     },
     leftIcon :{
-      options: ["user_tab","plus","pencil","edit"],
+      options: ["user_tab","plus_new","pencil","edit"],
```

ESLint

[error] Expected indentation of 12 spaces but found 6.

(indent)